presto-benchmark/src/main/java/io/prestosql/benchmark/AbstractOperatorBenchmark.java

```diff
@@ -181,7 +181,7 @@ protected final OperatorFactory createTableScanOperator(int operatorId, PlanNode
             public Operator createOperator(DriverContext driverContext)
             {
                 OperatorContext operatorContext = driverContext.addOperatorContext(operatorId, planNodeId, "BenchmarkSource");
-                ConnectorPageSource pageSource = localQueryRunner.getPageSourceManager().createPageSource(session, split, tableHandle, columnHandles);
+                ConnectorPageSource pageSource = localQueryRunner.getPageSourceManager().createPageSource(session, split, tableHandle, columnHandles, null);
```

Comment:
I think you can pass tuple domain all here

Reply:
Done.